src/ResourceManager/Sql/Commands.Sql/Az.Sql.psd1

```diff
@@ -286,10 +294,15 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = 'Initial Release with PowerShell and PowerShell Core Support'
+        ReleaseNotes = '* New Cmdlets for Management.Sql to allow customers to add TDE keys and set TDE protector for managed instances
+   - Add-AzSqlManagedInstanceKeyVaultKey
+   - Get-AzSqlManagedInstanceKeyVaultKey
+   - Remove-AzSqlManagedInstanceKeyVaultKey
+   - Get-AzSqlManagedInstanceTransparentDataEncryptionProtector
+   - Set-AzSqlManagedInstanceTransparentDataEncryptionProtector'
 
         # Prerelease string of this module
-        # Prerelease = ''
+        Prerelease = 'preview'
```

Comment:
@maddieclayton for `Az`, we're going with the semantic version pattern of `0.x.x` being preview (or unstable), and only roll out `-preview` versions for stable modules, so you can revert this change